### ISSUE TITLE
Error message

### ISSUE THREAD

**Issue by Dario-Rosa85** [ashthespy/Atom-LaTeX#250]
All the times I try to build a latex (I am using texlive) I get this error message, which nevertheless does not stop the compilation

Command Failed: pdflatex -synctex=1 -interaction=nonstopmode -file-line-error hopping_teo.tex

Anybody can help me?

**Comment by ashthespy** [comment]
That looks like a LaTeX compilation error and not a package error. You cal look click the "Show raw log" button to have a look.

**Comment by Dario-Rosa85** [comment]
Thank you. It looks like he has some problems with the bibtex:

> bibtex hopping_teo

This is BibTeX, Version 0.99d (TeX Live 2018/W32TeX)
The top-level auxiliary file: hopping_teo.aux
I found no \bibdata command---while reading file hopping_teo.aux
I found no \bibstyle command---while reading file hopping_teo.aux
(There were 2 error messages)


The point is that I do not have any bibliography in this small file. Can I build the latex without doing the bibtex step?
